stdlib/REPL/src/precompile.jl

```diff
@@ -31,11 +32,14 @@ UP_ARROW = "\e[A"
 DOWN_ARROW = "\e[B"
 
 repl_script = """
+Base.load_terminfo("xterm")
 2+2
 print("")
 printstyled("a", "b")
 display([1])
 display([1 2; 3 4])
+import REPL.banner
+banner()
```

Comment:
Why is this needed? Isn't the banner printed when the REPL is started?

Reply:
Hmm, I'll check whether this actually makes a difference and come back.